code/modules/ninja/suit/n_suit_verbs/ninja_adrenaline.dm

```diff
@@ -12,11 +11,17 @@
 		H.adjustStaminaLoss(-75)
 		H.stuttering = 0
 		H.reagents.add_reagent(/datum/reagent/medicine/amphetamine, 5)
```

Comment:
```suggestion
		H.reagents.add_reagent(/datum/reagent/medicine/amphetamine, 5)
		H.reagents.add_reagent(/datum/reagent/medicine/stabilizing_nanites, 1)
```

This is the only other suggestion I would have, but I'm not blocking for it. Stabilizing nanites cannot heal past 80 HP, so they will never bring the ninja back to full HP, but they can provide a critical boost toward getting out of damage slowdown in an emergency situation (and this button seems to be precisely for getting out of an emergency situation, since it only gets three uses)